### DISCUSSION TITLE
How can this service be free?

### DISCUSSION THREAD

**Discussion by tatlilimon** [xtekky/gpt4free#250] [General]
Openai charges a fee for api calls. You provide this service on your website without any charge. Are you rich enough to open your own api to the public xd

**Comment by tatlilimon** [comment]
I mean what is reverse engineered 3rd party api's ?

**Reply by paladenite** [reply]
From what I can tell the code itself it isn't an actual website itself, its an application on your computer that runs its own code. Reverse engineering this code is insane, i downloaded a zip file incase they shut this down, but im reading the code through github, and so far it seems legit and not scammy, but all it takes is a few lines of code to brick your computer so read the code first and download with care. The entire point of this project it seems is to make ai like chat gpt free before they start charging the fuck outta us to use it and so that everyone has an ai to use in the eventual ai wars.

**Reply by iscjesusmartinez** [reply]
it's the transformer it self? it's trained? actual size of the "core"? DalAI - LLAMA its about 120 GB and its a "nano GTP 3", now.. if you call this thing "free GPT", its the complete trained generative transformer? and the CONTROL API? actually, thing some deep about it... GPT 4 its a nano AGI, just for keep that on control and make that respond our questions, its NEEDED an four or 32 or 128 mini "gpt controller's" its like to control a leviatam, I don´t know, and it's "curious tha Open AI tell NOTHING about GTP4, but they publish a complete paper to GTP3 two years ago... I thing the code was "gifted" by some...one, or something, or maybe the future, or it's an non human but life bean and Open AI just put chains and energy to make it our slave :O, sorry 4 my russsian accent jaja.
so.... what kind of server I need to run this? the same that I use to run Crysis? XDXDXDXDXDD ba dum ps...come on, really, .... an 128 virtual cores? 256 GB of RAM? 10TB solid state? 1TB on redis? apache kafka? saludos a todos Y

**Comment by if8336** [comment]
This project just calls the third-party free api, not the gpt4 program. In order to attract users, many websites purchase a large number of API quotas from openai, and then give users several gpt4 quotas for free every day. Through this project, call those websites to access gpt4 indirectly, nothing more. In addition, openai makes money no matter who uses it, and violates the interests of those websites that are free for users to use.

**Reply by if8336** [reply]
Just like https://poe.com/
